File: Golang/Easy/3512_Minimum_Operations_to_Make_Array_Sum_Divisible_by_K/solution.go
package minimumoperationstomakearraysumdivisiblebyk

func minOperations(nums []int, k int) int {
	sumNum := 0
	for _, num := range nums {
		sumNum += num
	}

	count := 0
	for sumNum%k != 0 {
		count += 1
		sumNum -= 1
	}
	return count
}

File: Python/Easy/3512_Minimum_Operations_to_Make_Array_Sum_Divisible_by_K/solution.py
from typing import List

class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        sumNum = 0
        for num in nums:
            sumNum += num
        
        count = 0
        while sumNum % k != 0:
            count += 1
            sumNum -= 1

        return count